docker/tool-box/s2i/run

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# inspired by https://github.com/openshift/jenkins/blob/master/2/contrib/jenkins/jenkins-common.sh
+
+
+USER_ID=$(id -u)
+GROUP_ID=$(id -g)
+
+if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"997" ]; then
+  echo "tool-box:x:${USER_ID}:${GROUP_ID}:Tool Box User:${HOME}:/sbin/nologin" >> /etc/passwd
+fi
+
+tail -f /dev/null
```

Comment:
Is this considered a "best practice"? I know you need something to keep the container running, but I'm not sure if this is the best option

Reply:
specifically about `tail -f /dev/null` I am open to alternatives. A quick web search didn't provide anything else as simple and easy. I'd prefer not to run a webserver on the like.

Reply:
Oh, definitely not something so heavy... I had 2 thoughts when looking at this:

1. Perhaps something like `watch ps -a` so that you can easily see that the container is running and not frozen?
2. The other thought was I wonder if there might be an even lighter alternative?

Beyond that, I'm fine with this solution and so I will approve and we can revisit later.